src/exchanges/fetch.ts

```diff
@@ -75,9 +75,19 @@ const createFetchSource = (operation: Operation) => {
         abortController !== undefined ? abortController.signal : undefined,
     };
 
+    const startTime = Date.now();
     executeFetch(operation, fetchOptions).then(result => {
       if (result !== undefined) {
-        next(result);
+        next({
+          ...result,
+          operation: {
+            ...result.operation,
+            context: {
+              ...result.operation.context,
+              latency: Date.now() - startTime,
```

Comment:
This is something that I'd rather not have in `production` all the time, but I suppose it's ok to use this anyway :+1:

can we make sure that there's a default value of `0` in the `client`. Might not be nice, but we don't want people to actually  use this for something else